classes/Validate.php

```diff
@@ -638,8 +638,9 @@ public static function isBirthDate($date, $format = 'Y-m-d')
         if (!empty(DateTime::getLastErrors()['warning_count']) || false === $d) {
             return false;
         }
+        $twoHundredYearsAgo = time() - (200 * 365 * 24 * 60 * 60);
```

Comment:
```suggestion
        $twoHundredYearsAgo = new Datetime();
        $twoHundredYearsAgo->sub(new DateInterval('P200Y'));
```

Reply:
If you prefer use Datetime here, you need to update the whole method to be consistent :)

Reply:
done